web/ui/templates/service-discovery.html

```diff
@@ -18,7 +18,6 @@
 
     <h1>Service Discovery</h1>
     <div class="table-container">
```

Comment:
Do we still need to add this div/class? `table-container` seems only used to [add functionality and styles to tables](https://github.com/prometheus/prometheus/search?utf8=%E2%9C%93&q=table-container&type=).

Reply:
Indeed!